x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/create_notifications.ts

```diff
@@ -17,20 +17,19 @@ export const createNotifications = async ({
   ruleAlertId,
   interval,
   name,
-  tags,
```

Comment:
NOTE: Removed unused types such as `tags` and fixed code to look more like we would expect. Makes for a cleaner read of the code and easier understanding.